### PR TITLE
Bump tf to version 1.0

### PR DESCRIPTION
**Pull request by bwhaley**
Updates tests to use Terraform 1.0.

### CODE REVIEW

**Review by zackproser — APPROVED** [review]
LGTM

**Comment by yorinasub17** [comment]
Kicking off build prior to merge

**Comment by yorinasub17** [comment]
Ah looks like somehow the circleci config got broken along the way 🤦 . Can you update to remove the nightly trigger workflow?

**Comment by bwhaley** [comment]
Done. Mind triggering the tests again?

File: .circleci/config.yml [hunk 1]
```diff
@@ -55,14 +55,6 @@ workflows:
...
-    triggers:
-      - schedule:
-          cron: "0 0 * * *"
-          filters:
-            branches:
-              only:
-                - master
     jobs:
       - build
       - test:
```
Comment (yorinasub17) [inline .circleci/config.yml:60]: Ah looks like you forgot to remove this subkey of the `nightly-test` workflow.

**Comment by bwhaley** [comment]
Oh, whoops! Fixed.

**Comment by yorinasub17** [comment]
Looks like the build is going now!

**Review by yorinasub17 — APPROVED** [review]
Build passed! Going to merge this in.

**Comment by bwhaley** [comment]
Thank you! 🙏

**Pull request merged**